# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../isoquery/spell.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN isoquery-1.4/man/de.add{.orig,}
--- isoquery-1.4/man/de.add.orig        2011-04-15 21:19:33.600077822 +0200
+++ isoquery-1.4/man/de.add     2011-04-15 21:17:14.436077821 +0200
@@ -1,4 +1,4 @@
-PO4A-HEADER:mode=after;position=AUTOR;beginboundary=FakePo4aBoundary
+PO4A-HEADER:mode=after;position=AUTHOR;beginboundary=FakePo4aBoundary
 
 
 ÜBERSETZUNG
diff -ruN isoquery-1.4/man/pt.add{.orig,}
--- isoquery-1.4/man/pt.add.orig        2011-04-15 21:19:44.492077759 +0200
+++ isoquery-1.4/man/pt.add     2011-04-15 21:17:27.528077823 +0200
@@ -1,4 +1,4 @@
-PO4A-HEADER:mode=after;position=AUTOR;beginboundary=FakePo4aBoundary
+PO4A-HEADER:mode=after;position=AUTHOR;beginboundary=FakePo4aBoundary
 
 
 TRADUÇÃO

